sentry_sdk/sessions.py

```diff
@@ -10,23 +10,27 @@
 from sentry_sdk.utils import format_timestamp
 
 if MYPY:
-    from typing import Callable
-    from typing import Optional
     from typing import Any
+    from typing import Callable
     from typing import Dict
-    from typing import List
     from typing import Generator
+    from typing import List
+    from typing import Optional
+    from typing import Union
 
 
 def is_auto_session_tracking_enabled(hub=None):
-    # type: (Optional[sentry_sdk.Hub]) -> bool
+    # type: (Optional[sentry_sdk.Hub]) -> Union[Any, bool, None]
```

Comment:
you sure this shouldn't have stayed `bool`?

perhaps a `return bool(should_track)` can avoid mypy errors